### PR TITLE
Requirement Updates

### PR DESCRIPTION
**Pull request by jeff-hernandez**
This updates requirements to later versions.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,8 +3,8 @@ pandas>=0.23.0
 tqdm>=4.19.2
```
Comment (rwedge) [inline requirements.txt]: Do we use tqdm directly in this library?

Comment (jeff-hernandez) [reply requirements.txt]: It doesn't seem we do. Should I remove?

Comment (rwedge) [reply requirements.txt]: Yeah let's remove

Comment (jeff-hernandez) [reply requirements.txt]: removed

**Pull request merged**